README.md

```diff
@@ -9,24 +9,23 @@ The models comprise 27 compartments, including nine pregnancy-specific compartme
 * The models are provided as ready-to-use MoBi projects (subfolder _Models_)
   * _Cefazolin_, _Cefuroxime_ and _Cefradine_ model evaluation is described in [[3](#references)]
   * _Caffeine_, _Midazolam_, _Nifedipine_, _Metoprolol_, _Ondansetron_, _Granisetron_, _Diazepam_ and _Metronidazole_ model evaluation is described in [[1](#references)]
-  
-* _BuildingBlocks_ subfolder contains MoBi building block for spatial structure and passive transports. Those building blocks can be used in MoBi to build new substance models for pregnant women as described in the next section.
 
```

Comment:
Please keep "Building blocks subfolder..." here and don't move to the Howto section.
Here the repository content is described